raybot/commands/whatdidiorder.rb

```diff
@@ -7,31 +7,37 @@ class WhatDidIOrder < SlackRubyBot::Commands::Base
       match (/^.*(whatdidiorder).*$/) do |client, data, match|
         samaya = Samaya.new
 
-        my_name = client.store.users[data.user]["profile"]["last_name"]
-        break unless my_name
+        name = client.store.users[data.user]["profile"]["last_name"]
+        break unless name
```

Comment:
probably worth some sort of error message here so ppl know why it's not working?

Reply:
Done. This also apparently didn't work before cause "" is truthy in ruby.